src/main/java/io/apimatic/core/utilities/CoreHelper.java

```diff
@@ -1187,7 +1187,6 @@ private static String getAccessorStringFormat(
                 return "%7C";
             case TSV:
                 return "%09";
```

Comment:
Nits, Non-blocker: Use proper constant variable name for the hardcoded strings for CSV, PSV and TSV to increase the readability.

Reply:
Added the proper constants for hardcoded strings